Sprint-1/errors/3.js

```diff
@@ -1,4 +1,4 @@
-const cardNumber = 4533787178994213;
+const cardNumber = "4533787178994213";
```

Comment:
The exercise asks you to update "the expression last4Digits is assigned to, in order to get the correct value"

Can you try doing this? In your code you have changed the data type to a string. Can you think why or when it might not be a good idea to change a number to a string?

Reply:
If cardNumber were a number (not a string), this would raise a TypeError because numbers don't have a slice() method.
If cardNumber were a number, JavaScript would throw a TypeError because slice() is not a method for numbers.
 Why String Conversion Might Not Always Be Ideal:
 1-Loss of Numerical Context: After converting to a string, performing arithmetic operations requires converting back to a number.
 2-Performance Considerations: String manipulation can be less efficient than numerical operations, especially in large-scale data processing.
 3-Clarity: Maintaining a variable as a number can make the code easier to understand in contexts where numeric operations are expected. ​​